accounts/forms.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+class SignUpForm(UserCreationForm):
+
+    class Meta:
+        model = User
+        fields = ('username', 'email', 'password1', 'password2')
```

Comment:
```suggestion
        fields = ('username', 'email',)
```
UserCreationForm継承していると上記の書き方でもpassword1とpassword2をfieldsに保持してくれるようです。

Reply:
修正しました。